src/v3/src/transformer/layout/safeMode/transformSafeModePoll.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {
+  DescriptionElement,
+  IdxStepTransformer,
+  IStepperContext,
+  SpinnerElement,
+  StepperNavigatorElement,
+  TitleElement,
+  UISchemaElement,
+  UISchemaLayout,
+  UISchemaLayoutType,
+} from '../../../types';
+import { generateRandomString, loc } from '../../../util';
+
+export const transformSafeModePoll: IdxStepTransformer = ({
+  transaction,
+  formBag,
+}) => {
+  const { uischema } = formBag;
+
+  const titleElement: TitleElement = {
+    type: 'Title',
+    options: {
+      content: loc('poll.form.title', 'login'),
+    },
+  };
+
+  const pollIntervalMs = transaction.nextStep?.refresh;
+
+  if (typeof pollIntervalMs !== 'number') {
+    return formBag;
+  }
+
+  const pollIntervalSec = Math.ceil(pollIntervalMs / 1000);
+
+  const descriptionElement: DescriptionElement = {
+    type: 'Description',
+    contentType: 'subtitle',
+    options: {
+      content: loc(
+        'poll.form.message',
+        'login',
+        [pollIntervalSec],
+        { $1: { element: 'span' } },
```

Comment:
🥳 Love that this is getting usage 🙏

Reply:
Also minor nit, should this have the `strong` class added to bold the number?

Reply:
I _believe_ we generally don't want to use bold text within strings any more to emphasize text. @alexdahl-okta @narayun-okta can you confirm?

Reply:
Good question!

If we've landed on a static string (the number doesn't count down), then we probably don't need bold (we don't need to bring attention to a changing value).

I don't think we have a blanket rule against bold, and some draft guidelines from content design have a least one case where bold is recommended. Bold is a helpful tool in the toolbox designers would like to use sometimes.

But I believe there is globalization pressure against bold – it's not native to all writing systems, and makes translation more difficult. So do agree that we can avoid bold unless we have a compelling case for emphasis.